src/main/kotlin/org/rust/lang/core/resolve2/FacadeMetaInfo.kt

```diff
@@ -43,3 +44,9 @@ fun RsMod.getDirectoryContainedAllChildFiles(): VirtualFile? {
     val (_, _, modData) = getModInfo(this) as? RsModInfo ?: return null
     return PersistentFS.getInstance().findFileById(modData.directoryContainedAllChildFiles)
 }
+
+fun findPreludeUsingNewResolve(element: RsElement): RsMod? {
```

Comment:
nit: Do we actually need this public function here?
Maybe move it into `NameResolution.kt` and make private?